make/build-support/version-numbers

```diff
@@ -23,7 +23,7 @@
 # questions.
 #
 
-JTREG_VERSION=7.5
+JTREG_VERSION=7.6
```

Comment:
Hello Christian, is it intentional that this version says `7.6` and the "unreleased" link in the changelog for the next release uses `7.5+1`?

Reply:
Yes. `7.6` here indicates the new version for the upcoming release(s) and `7.5+1` in the changelog refers to git tags for detailed comparisons.

Reply:
The `+1` vs `.1` has always confused me. When I read that diff I thought it represented 7.5.1.